swift/apple/FirezoneNetworkExtension/PacketTunnelProvider.swift

```diff
@@ -93,23 +117,30 @@ class PacketTunnelProvider: NEPacketTunnelProvider {
       let internetResourceEnabled =
         enabled != nil ? enabled == "true" : (tunnelConfiguration?.internetResourceEnabled ?? false)
 
-      let adapter = Adapter(
-        apiURL: apiURL,
-        token: token,
-        id: id,
-        logFilter: logFilter,
+      // Create and start the adapter
+      let adapter = AdapterUniFfi(packetTunnelProvider: self)
+      self.adapter = adapter
+
+      try adapter.start(
+        apiUrl: apiURL,
+        token: token.description,
```

Comment:
Curious why `.description` was appended here? Did the type change from String, or I guess that's due to the new UniFFI type?

Reply:
`Token` was always the underlying type, but FFI needs String eventually - I can still pass `Token` here and move the conversion at the `Session.newIos()`, it's a bit more clear then.